incubator/elasticsearch/values.yaml

```diff
@@ -5,12 +5,15 @@ appVersion: "5.4"
 
 image:
   repository: "docker.elastic.co/elasticsearch/elasticsearch-oss"
-  tag: "6.0.0"
+  tag: "6.0.1"
```

Comment:
Another thing I noticed with our deploy was that Kibana 5.4.3 complained that not all parts of our Elk stack were at the same version. I am not sure that's still the case with the 6.x series.